backend/src/main/java/ai/verta/modeldb/versioning/blob/diff/DiffComputer.java

```diff
@@ -305,14 +305,14 @@ public static AutogenPythonEnvironmentDiff computePythonEnvironmentDiff(
                     a,
                     b,
                     AutogenPythonEnvironmentBlob::getConstraints,
-                    AutogenPythonRequirementEnvironmentBlob::getLibrary,
+                    e -> e.getLibrary() + e.getConstraint(),
```

Comment:
@conradoverta : do the changes make sense to you? If Ihave different hashing in differ and merger then we have issues during merge around modified  being detected as added

Reply:
generates different key for the map